### PR TITLE
MAT-1062 Using fhir rules to validate fhir library names.

### PR DESCRIPTION
**Pull request by cday-semanticbits**

### CODE REVIEW

File: src/main/java/mat/client/cqlworkspace/CQLStandaloneWorkSpacePresenter.java [hunk 1]
```diff
@@ -75,7 +75,8 @@
     private SimplePanel emptyWidget = new SimplePanel();
     private boolean isCQLWorkSpaceLoaded = false;
     private final CQLLibraryServiceAsync cqlService = MatContext.get().getCQLLibraryService();
-    private static final String ERROR_INCLUDE_ALIAS_NAME_NO_SPECIAL_CHAR = "Invalid Library Alias. Must be unique, start with an alpha-character or underscore followed by an alpha-numeric character(s) or underscore(s), and must not contain spaces.";
+    private static final String FHIR_ERROR_INCLUDE_ALIAS_NAME_NO_SPECIAL_CHAR = "Invalid Library Alias. Must be unique, start with an upper case letter followed by an alpha-numeric character(s) or underscore(s), and must not contain spaces.";
+    private static final String QDM_ERROR_INCLUDE_ALIAS_NAME_NO_SPECIAL_CHAR = "Invalid Library Alias. Must be unique, start with an alpha-character or underscore followed by an alpha-numeric character(s) or underscore(s), and must not contain spaces.";;
```
Comment (jkotanchik-SB) [inline src/main/java/mat/client/cqlworkspace/CQLStandaloneWorkSpacePresenter.java:79]: extra trailing semi-colon.

Comment (jkotanchik-SB) [inline src/main/java/mat/client/cqlworkspace/CQLStandaloneWorkSpacePresenter.java:79]: extra trailing semi-colon.

Comment (adongare) [reply src/main/java/mat/client/cqlworkspace/CQLStandaloneWorkSpacePresenter.java:79]: 👀

Comment (cday-semanticbits) [reply src/main/java/mat/client/cqlworkspace/CQLStandaloneWorkSpacePresenter.java:79]: will fix.

**Pull request merged**